### ISSUE TITLE
ES Explore: select records from map

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#948]
The ES Explore all records page allows you to click on one of the dots on the map to see the associated records, but this behaviour seems inconsistent, and produces different results depending on how far you are zoomed in.

Is there a logic here that I'm not understanding?

Is it possible to reinstate the ability to trace over a set of dots on the map and see all the records under those dots?

**Comment by johnvanbreda** [comment]
Agreed, this area needs work. One of the issues is Leaflet doesn't seem to have an easy way of setting up a "marquee" tool for record selection.
@burkmarr do you have any experience in this area?

**Comment by burkmarr** [comment]
I haven't done it @johnvanbreda. The core of Leaflet seems to be quite light-weight and relies on plugins for lots of stuff. I did a quick google on it and there seem to be a few plugins that might contribute to a solution.

**Comment by kitenetter** [comment]
Two issues - inconsistencies and marquee

**Comment by kitenetter** [comment]
Actions:
@kitenetter to document inconsistencies with current single-click functionality.

Review options for adding a marquee select tool in future.

Current workaround is for users to create a filter that searches by grid sqaure or polygon.
